jolt/Cargo.toml

```
@@ -28,6 +29,7 @@ sha2-guest = { path = "./sha2-guest" }
 sha3-guest = { path = "./sha3-guest" }
 fibonacci-guest = { path = "./fibonacci-guest" }
 bigmem-guest = { path = "./bigmem-guest" }
+ecdsa-guest = {path = "./ecdsa-guest"}
 
```

そのままだとgetrandomのコンパイルエラーで落ちる。明示的にcustomフィーチャを指定するとパスした。

```suggestion
ecdsa-guest = {path = "./ecdsa-guest"}
getrandom = { version = "0.2", features = ["custom"] }
```

---

https://github.com/a16z/jolt/issues/399